plugins/CMakeLists.txt

```diff
@@ -23,6 +23,13 @@ if(BUILD_PLUGIN_ICEBRIDGE)
     add_subdirectory(icebridge)
 endif()
 
+if(BUILD_PLUGIN_HDF)
+    if (NOT PDAL_HAVE_LIBXML2)
+        message(FATAL_ERROR "Can't build hdf plugin without libxml2")
```

Comment:
HDF driver should not require libxml2. That was true of the Icebridge driver for its metadata thing. I think this can be removed.